doc/modules/mod_commands.md

```diff
@@ -0,0 +1,72 @@
+# MongooseIM's command set
+
+## Purpose
+
+This is a basic set of administration and client commands, which provide sort of an API for MongooseIM.
+The goal is to provide a consistent, easy to use API, providing enough information to allow for auto-generation of access methods.
+It is currently used to provide REST API, both admin and client interface, and in the future may replace current implementation of mongooseimctl.
+
+
+## Configuration
+
+This module contains command definitions which are loaded when the module is activated.
+There are no more configuration parameters, so the following entry in config file is sufficient:
```

Comment:
There are no more configuration parameters, so the following entry in the config file is sufficient: